CMakeLists.txt

```diff
@@ -41,10 +41,10 @@ include(cmake/UnitTest.cmake)
 include(cmake/LYTestWrappers.cmake)
 include(cmake/Monolithic.cmake)
 include(cmake/SettingsRegistry.cmake)
-include(cmake/TestImpactFramework/LYTestImpactFramework.cmake)
 include(cmake/CMakeFiles.cmake)
 include(cmake/O3DEJson.cmake)
 include(cmake/Subdirectories.cmake)
+include(cmake/TestImpactFramework/LYTestImpactFramework.cmake)
```

Comment:
We should probably add a comment saying that we put this dependency at the end because nothing else depends on it.